### PR TITLE
Create django skeleton

### PR DESCRIPTION
**Pull request by rudigiesler**
Create a django skeleton for the API, and link all the required services (travis, coverage, requires.io) to the repo, and add a readme with testing and running instructions.

### CODE REVIEW

**Comment by rudigiesler** [comment]
Not sure what coveralls is still waiting for, but ready for review. @justinvdm

File: README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+[![Build Status](https://travis-ci.org/praekelt/seed-auth-api.svg?branch=develop)](https://travis-ci.org/praekelt/seed-auth-api)
+[![Requirements Status](https://requires.io/github/praekelt/seed-auth-api/requirements.svg?branch=feature%2Fissue-2-create-django-skeleton)](https://requires.io/github/praekelt/seed-auth-api/requirements/?branch=feature%2Fissue-2-create-django-skeleton)
+
+# seed-auth-api
+Seed Auth API. User and permissions store, authentication and authorization.
+
+## Running
+
+ * `pip install -e .`
+ * `psql -U postgres -c "CREATE DATABASE seed_auth;`
```
Comment (justinvdm) [inline README.md]: Wondering if we will need to a line here for enabling the hstore extension, but happy for us to add that when we need it.

Comment (rudigiesler) [inline README.md]: Is that something that is done here? I thought the migrations might do it for us.. But I will have to try things and see what works. So I'm 👍  on adding it when we need it, so we can test if we actually need it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+ * `./manage.py migrate`
+ * `./manage.py createsuperuser`
+ * `./manage.py runserver --settings=seed_auth_api.testsettings`
```
Comment (justinvdm) [inline README.md]: Wondering whether our dev environment settings would ever differ from test settings (in which case maybe we need a dev settings module).

File: README.md (hunk 1 above)
Comment (justinvdm) [inline README.md]: Ah, probably better done with a migration. Cool, lets add it if we need it.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+[![Coverage Status](https://coveralls.io/repos/github/praekelt/seed-auth-api/badge.svg?branch=develop)](https://coveralls.io/github/praekelt/seed-auth-api?branch=develop)
+[![Build Status](https://travis-ci.org/praekelt/seed-auth-api.svg?branch=develop)](https://travis-ci.org/praekelt/seed-auth-api)
+[![Requirements Status](https://requires.io/github/praekelt/seed-auth-api/requirements.svg?branch=feature%2Fissue-2-create-django-skeleton)](https://requires.io/github/praekelt/seed-auth-api/requirements/?branch=feature%2Fissue-2-create-django-skeleton)
```
Comment (justinvdm) [inline README.md]: Should this change to the `develop` branch?

File: README.md (hunk 2 above)
Comment (rudigiesler) [inline README.md]: Maybe if we get do get to a point where it differs, then we can create a dev setting module?

Comment (justinvdm) [inline README.md]: Cool, lets add it when we need it.

**Comment by justinvdm** [comment]
Maybe coveralls is't completing its check since it doesn't have anything to compare with.

**Comment by justinvdm** [comment]
One minor comment, looks good!

**Comment by rudigiesler** [comment]
@justinvdm Fixed the branch reference in the requires.io tag. Ready for another review.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**